src/helpers/time.ts

```diff
@@ -0,0 +1,21 @@
+import ms from "ms";
+import { getLogger } from "../bindings";
+
+const MILLISECONDS = 1000;
+
+export const convertTimeToSeconds = (time: string): number => {
+  let _time = time;
+
+  // `ms` doesn't support month conversion, so converting to days from months.
+  if (_time.toLowerCase().includes("month")) {
+    _time = "30 Days";
```

Comment:
Not every month is 30 days but there's always 12 months in a year. Maybe that's the best approach?

Reply:
So you're suggesting that, if take average of a month? `30.4 days` as a month?

Reply:
325.25 days in a year

Reply:
I think it 365.25?!